app/models/program_session.rb

```diff
@@ -60,29 +60,6 @@ def self.create_from_proposal(proposal)
     end
   end
 
-  def self.create_draft_from_proposal(proposal)
```

Comment:
Not sure if this was some weird PR artifact but I see this method has been removed and create_from_proposal referenced but I don't see it listed in the PR anywhere.  It worked when I tested the app so maybe it got lost in another commit that's not in the PR?

Reply:
I did remove create_draft_from_proposal, and call create_from_proposal everywhere. Since I am now not always creating a draft, but handling it differently if the proposal is waitlisted, I thought create_draft_from_proposal was now poorly named, and since create from proposal was only being called in a test, I repurposed it to create a program session with the proper status according to the proposal status.